src/scenes/AwaitingConnectionScene.tsx

```diff
@@ -85,13 +127,7 @@ function CopyButton({ connectorUri }: { connectorUri?: string }): JSX.Element {
     'Copy QR code'
```

Comment:
@michaelfurutin Please, add some simple opacity hover for this button on mobile

Reply:
added